cluster.tf

```diff
@@ -1,104 +1,53 @@
-##############################################################################
-# Find valid IKS/ROKS Cluster versions for region
-##############################################################################
-
-data "ibm_container_cluster_versions" "cluster_versions" {}
-
-##############################################################################
-
-
 ##############################################################################
 # Cluster Locals
 ##############################################################################
 
 locals {
-  worker_pools_map = module.dynamic_values.worker_pools_map # Convert list to map
-  clusters_map     = module.dynamic_values.clusters_map     # Convert list to map
-  latest_kube_version = {
-    openshift = "${data.ibm_container_cluster_versions.cluster_versions.valid_openshift_versions[length(data.ibm_container_cluster_versions.cluster_versions.valid_openshift_versions) - 1]}_openshift"
-    iks       = data.ibm_container_cluster_versions.cluster_versions.valid_kube_versions[length(data.ibm_container_cluster_versions.cluster_versions.valid_kube_versions) - 1]
-  }
+  clusters_map = module.dynamic_values.clusters_map # Convert list to map
 }
 
 ##############################################################################
-
-
-##############################################################################
-# Create IKS/ROKS on VPC Cluster
+# Create ROKS on VPC Cluster
 ##############################################################################
 
-resource "ibm_container_vpc_cluster" "cluster" {
+
+module "cluster" {
+  depends_on = [
+    module.vpc
+  ]
   for_each          = local.clusters_map
-  name              = "${var.prefix}-${each.value.name}"
-  vpc_id            = each.value.vpc_id
+  source            = "git::https://github.com/terraform-ibm-modules/terraform-ibm-base-ocp-vpc.git?ref=v2.1.1"
```

Comment:
I tested with the following configuration - 
```
"clusters": [
    {
      "cos_name": "cos",
      "entitlement": null,
      "kube_type": "openshift",
      "kube_version": "default",
      "machine_type": "bx2.16x64",
      "name": "workload-cluster",
      "resource_group": "workload-rg",
      "kms_config": {
        "private_endpoint": true,
        "crk_name": "slz-roks-key"
      },
      "subnet_names": [
        "vsi-zone-1",
        "vsi-zone-2"
      ],
      "update_all_workers": false,
      "vpc_name": "workload",
      "worker_pools": [
        {
          "entitlement": "cloud_pak",
          "flavor": "bx2.16x64",
          "name": "default",
          "subnet_names": [
            "vsi-zone-1",
            "vsi-zone-2"
          ],
          "vpc_name": "workload",
          "workers_per_subnet": 1
        }
      ],
      "workers_per_subnet": 1
    }
  ]
```

I would expect the "default" (ibm_container_vpc_cluster) be from the high level information and the additional worker pool from the worker_pools information.  In this case, I expect 2 pools from this configuration